src/popover-helpers.ts

```diff
@@ -84,7 +84,6 @@ function topMostAutoPopover(document: Document): HTMLElement | null {
   const documentPopovers = autoPopoverList.get(document);
   for (const popover of documentPopovers || []) {
     if (!popover.isConnected) {
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Should we be concerned about disabling this check?

Reply:
I'm not sure why these got added... this rule is off by default 🤔